packages/cli/tests/compile/integration.test.ts

```diff
@@ -140,6 +140,21 @@ test('malformed ICU message json', async () => {
   ).rejects.toThrowError('SyntaxError: Expected "," but end of input found.')
 }, 20000)
 
+test('skipped malformed ICU message json', async () => {
+  await expect(
+    exec(
+      `${BIN_PATH} compile  --skip-errors ${join(
+        __dirname,
+        'lang/malformed-messages.json'
+      )}`
+    )
+  ).resolves.toMatchSnapshot({
+    stderr: expect.stringMatching(
+      /^Error validating message "my name is {name" with ID "a1dd2" in file .*\/packages\/cli\/tests\/compile\/lang\/malformed-messages.json/
+    ),
+  } as Partial<any>)
```

Comment:
This type was being difficult, it seems to think we have not unwrapped the promise yet. 

Let me know if this is an issue and I can spend more time here, I didn't think it added a lot of value since the string matching will fail if anything weird happens.